src/web/core/src/main/java/org/geoserver/web/admin/StatusPage.java

```diff
@@ -198,6 +201,7 @@ private void updateModel() {
         values.put(KEY_COVERAGEACCESS_KEEP_ALIVE_TIME, Integer.toString(coverageAccess.getKeepAliveTime()));
 
         values.put(KEY_UPDATE_SEQUENCE, Long.toString(geoServerInfo.getUpdateSequence()));
+        values.put(KEY_JAVA_RENDERER, sun.java2d.pipe.RenderingEngine.getInstance().getClass().getName());
```

Comment:
There is a chance this will break on a non Oracle VM (IBM, maybe Azul), maybe move it into a method, guard with try/catch, and report N/A if it cannot be found?